source/includes/cloudstack/_volumes.md

```diff
@@ -234,3 +234,71 @@ Optional | &nbsp;
 ------ | -----------
 `name`<br/>*string* | A ***unique name*** to be given to the newly created **snapshot**. If this parameter is not provided then by default the concatenation of the *instance name*, *volume name* and the *current timestamp* is used. <br/><br/>*Eg:*<br/>&nbsp;&nbsp;&nbsp;&nbsp;*[instance.name]\_[volume.name]\_[timestamp]*<br/>&nbsp;&nbsp;&nbsp;&nbsp;***i-root-6E7_RapidVol_20190117153537***
 `rapid`<br/>*boolean* | Indicates the ***location*** as to where the snapshot is supposed to be made. <br/><br/>Setting this to **true** will ensure that the snapshot is created in the same primary storage as where the volume is. If **false**, then the snapshot is created in a secondary storage. <br/><br/>*Note: Rapid snapshots enable much faster volume and template creation than from regular snapshots, but at a higher expense. Not all volumes support the* **rapid** *snapshot option.*
+
+#### Resize volumes 
+
+***To resize a data disk / volume:***
+```shell
+
+curl --request POST \
+  --url 'https://cloudmc_endpoint/api/v2/services/cloudstack-aaaa/acs-env-1/volumes/ed30cd74-3e83-473c-9fe3-6b1cb1652f85?operation=resize' \
+  --header 'Content-Type: application/json' \
+  --header 'mc-api-key: your_api_key' \
+  --data 'request-body'
+```
+> Request body example:
+
+```json
+{
+	"sizeInGb": 12,
+	"diskOfferingId": "fd78763c-f33a-43f3-b1e3-63bf59a48350"
```

Comment:
we need to have the disk offering? I though this was fetched from the existing disk

Reply:
This is how the api has been designed.. I have just updated the doc based on the existing behaviour for data disks

Reply:
Ok.